7164 update fips enablement documentation add info (#7181)
OpenLiberty/docs: modules/ROOT/pages/enable-fips.adoc

----
certutil -L -d /etc/pki/nssdb
----
+

> In FIPS mode, Semeru Runtimes does not support the usage of the NSS database with a password. When the PKCS#12 file-based keystore is being imported into the NSS database, if the `pk12util` tool prompts to set up the password, press the Enter key twice to set up an empty password.

Reviewer: ```suggestion
In FIPS mode, Semeru Runtimes does not support the usage of the NSS database with a password. If the `pk12util` tool prompts you to set up a password when you import the PKCS#12 file-based keystore, press Enter twice to set up an empty password.

```
